fastplotlib/tools/_tooltip.py

```diff
@@ -0,0 +1,298 @@
+        return self._world_object
+
+    @property
+    def font_size(self):
+        """Get or set font size"""
+        return self._text.font_size
+
+    @font_size.setter
+    def font_size(self, size: float):
+        self._text.font_size = size
+
+    @property
+    def text_color(self):
+        """Get or set text color using a str or RGB(A) array"""
+        return self._text.material.color
+
+    @text_color.setter
+    def text_color(self, color: str | tuple | list | np.ndarray):
+        self._text.material.color = color
+
+    @property
+    def background_color(self):
+        """Get or set background color using a str or RGB(A) array"""
+        return self._plane.material.color
+
+    @background_color.setter
+    def background_color(self, color: str | tuple | list | np.ndarray):
+        self._plane.material.color = color
+
+    @property
+    def outline_color(self):
+        """Get or set outline color using a str or RGB(A) array"""
+        return self._line.material.color
+
+    @outline_color.setter
+    def outline_color(self, color: str | tuple | list | np.ndarray):
+        """Get or set outline color using a str or RGB(A) array"""
+        self._line.material.color = color
+
+    @property
+    def padding(self) -> np.ndarray:
+        """
+        Get or set the background padding in number of pixels.
+        The padding defines the number of pixels around the tooltip text that the background is extended by.
+        """
+
+        return self.padding[0, :2].copy()
+
+    @padding.setter
+    def padding(self, padding_xy: tuple[float, float]):
+        self._padding[0, :2] = padding_xy
+        self._padding[1, :2] = -np.asarray(padding_xy)
+
+    def _set_position(self, pos: tuple[float, float]):
+        """
+        Set the position of the tooltip
+
+        Parameters
+        ----------
+        pos: [float, float]
+            position in screen space
+
+        """
+        # need to flip due to inverted y
+        x, y = pos[0], pos[1]
+
+        # put the tooltip slightly to the top right of the cursor positoin
+        x += 8
+        y -= 8
+
+        self._text.world.position = (x, -y, 0)
+
+        bbox = self._text.get_world_bounding_box() - self._padding
+        [[x0, y0, _], [x1, y1, _]] = bbox
+
+        self._plane.geometry.positions.data[masks.x0] = x0
+        self._plane.geometry.positions.data[masks.x1] = x1
+        self._plane.geometry.positions.data[masks.y0] = y0
+        self._plane.geometry.positions.data[masks.y1] = y1
+
+        self._plane.geometry.positions.update_range()
+
+        # line points
+        pts = [[x0, y0], [x0, y1], [x1, y1], [x1, y0], [x0, y0]]
+
+        self._line.geometry.positions.data[:, :2] = pts
+        self._line.geometry.positions.update_range()
+
+    def _event_handler(self, custom_tooltip: callable, ev: pygfx.PointerEvent):
+        """handles the tooltip appear event, determines the text to be set in the tooltip"""
```

Comment:
```suggestion
        """Handles the tooltip appear event, determines the text to be set in the tooltip"""
```
Also, always capitalizing doc strings